tests/js/client/restart/test-restart-cluster.js

```diff
@@ -86,6 +86,27 @@ function testSuite() {
   };
 
   return {
+    tearDownAll : function() {
+      // Need to restart without authentication for other tests to succeed:
+      let coordinators = getServers('coordinator');
+      let coordinator = coordinators[0];
+      let instanceInfo = global.instanceInfo;
+      let newInstanceInfo = {
+        arangods: [ coordinator ],
+        endpoint: instanceInfo.endpoint,
+      };
+      let options = global.testOptions;
+      let shutdownStatus = pu.shutdownInstance(newInstanceInfo, options, false); 
+      coordinator.pid = null;
+      console.warn("Restarting coordinator without authentication as cleanup...", coordinator);
```

Comment:
I found the message hard to understand, and had to think about the "without authentication as cleanup" for a long time :sweat_smile: 
```suggestion
      console.warn("Cleaning up and restarting coordinator without authentication...", coordinator);
```